### PR TITLE
Added travis CI

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
In this pull request I added travis CI tests to this Rep and squashed all the commits into one.
---------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+ - "3.6"
+install:  
+ - pip install -r requirements.txt
+script:
+  - python test.py
```
Comment (aashutoshrathi) [inline .travis.yml]: Make Travis more like [this](https://github.com/aashutoshrathi/labper/blob/master/.travis.yml)
Add random values to all env variables we have for this repo. And luckily we are using the same config as that repo of mine, i.e. we are using Postgres for the server.
So, keep Postgres and it's script.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Certificate Generator Server
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/06bd6acff9cd4506985192596642ef5f)](https://www.codacy.com/app/JBossOutreach/certificate-generator-server?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JBossOutreach/certificate-generator-server&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/OussEmaDevCode/certificate-generator-server.svg?branch=master)](https://travis-ci.org/OussEmaDevCode/certificate-generator-server)
```
Comment (aashutoshrathi) [inline README.md]: We show Build Status of main repository's master branch.
So, please replace all `OussEmaDevCode` with `JBossOutreach` in Line 3

File: test.py [hunk 3]
```diff
@@ -0,0 +1 @@
+print("Hello world!")
```
Comment (aashutoshrathi) [inline test.py]: We don't need this file, but even if you add tests, make sure you assert something.

File: .travis.yml (hunk 1 above)
Comment (jatinkatyal13) [reply .travis.yml]: We even need to run tests so we should add that too !

File: README.md (hunk 2 above)
Comment (OussEmaDevCode) [reply README.md]: I can't replace it I am not the owner of the main rep I can only do it to the fork

**Comment by aashutoshrathi** [comment]
Thanks for the PR :smile:

**Pull request closed without merging**